test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -398,6 +398,17 @@ if (!browserUtils.isFirefox) {
     });
 }
 
+if (eventUtils.hasPointerEvents) {
+    test('pointer down', function () {
```

Comment:
Test checks that `pointerdown` event are simulated in all browsers which have `pointer` events behavior.